benchmark/domain/synthetic_3.py

```diff
@@ -0,0 +1,92 @@
+"""Synthetic dataset. Custom parabolic test with irrelevant parameters."""
+
+from uuid import UUID
+
+from numpy import pi, sin, sqrt
+from pandas import DataFrame
+
+from baybe.campaign import Campaign
+from baybe.objective import SingleTargetObjective
+from baybe.parameters import NumericalContinuousParameter, NumericalDiscreteParameter
+from baybe.recommenders import RandomRecommender
+from baybe.searchspace import SearchSpace
+from baybe.simulation import simulate_scenarios
+from baybe.targets import NumericalTarget, TargetMode
+from benchmark.definition import Benchmark
+
+
+def lookup_synthetic_3(x: float, y: float, z: int) -> float:
```

Comment:
Since x and y are assumed to be within a certain range, this should also be checked/verified inside this function.

Reply:
This might also allow you to get rid of the unnecessary `return 0` if you also assert certain values of `z`.